### ISSUE TITLE
Bug: Unexpected behavior when iris is closed allows player to survive

### ISSUE THREAD

**Issue by Hunter24681** [Tau-ri-Dev/JSGMod-1.12.2#230]
**Describe the bug**
When moving from an interdimensional stargate to a gate with a closed iris, the iris doesn't kill the player, but does leave them in a non-standard state.  New outgoing wormholes, nether portals, and betweenlands portals will not transport the player.  Aroma1997 dimensional portals still work- and reset the player state, as does logging out/back into the world.

**To Reproduce**
1. Set up an interdimensional gate (I used end -> overworld)
2. Set overworld gate iris to close on incoming wormhole
3. Chunkload overworld gate (I used ftbUtils, not sure if this is necessary)
4. Dial overworld gate from other gate and go through without entering GDO code

**Expected behavior**
Player should be killed by closed iris.

**Screenshots**
Nothing stands out in the console, but perhaps the tick lag is related?
![image](https://user-images.githubusercontent.com/76822995/221071556-f063d38d-d9e4-4cd1-8806-900335a6e047.png)

**Mod version**
jsg-1.12.2-4.11.0.5-client.jar

**Additional context**
Iris impact sound plays, suggesting that the player does come into contact with it.
Removed all modded armor/baubles to ensure they were not preventing death.
Gates on same dimension do not exhibit this behavior.

**Comment by Hunter24681** [comment]
Video to confirm behavior since it wasn't reproducible on dev end.

https://user-images.githubusercontent.com/76822995/221276086-dc68e8d8-d357-4bd2-9a38-371851b078d2.mp4

**Comment by MineDragonCZ** [comment]
> Video to confirm behavior since it wasn't reproducible on dev end. reviewing it there is a damage tick that happens. I don't think it's the scaling health health increase though, since same-dimension closed iris is still deadly.

You have some special thing on you (from other mod) that allows you to go through the iris. (The iris hits you, but does not kill you)

**Comment by Hunter24681** [comment]
@MineDragonCZ I don't think this is the case.

First, in the setup I attached above, even all those items on me, going from an overworld gate to another overworld gate with a closed iris kills me.

Second, I was able to reproduce it on a new save.  No health upgrades, no inventory, baubles, or any sort of equipment. I also set up a tinkers smeltery with a lot of casting basins to create heavy block entity tick lag on reloading the dimension.  In this situation, the interdimensional gate causes the above issue, but another gate on the overworld kills me as expected.

New world, no equipment, interdimensional gate:
https://user-images.githubusercontent.com/76822995/221375125-55837dc2-a8b2-4dbe-8824-da0e8abf0c67.mp4

Console output, note the tick lag
![image](https://user-images.githubusercontent.com/76822995/221375142-d7be3922-5c37-48ec-8998-69b7d6f58276.png)

Same save, no equipment changes, overworld to overworld:
https://user-images.githubusercontent.com/76822995/221375163-66e2dace-0754-4e62-893f-604235bed8fc.mp4


I am fairly confident the death by iris is being skipped in the skipped ticks.
